teckers/teckers/MessageCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BaseCell.swift
+//  teckers
+//
+//  Created by Maggie Mendez on 8/24/19.
+//  Copyright © 2019 Teckers. All rights reserved.
+//
+
+import UIKit
+
+class MessageCell: UITableViewCell {
+    
+    @IBOutlet weak var imageUser: UIImageView!
+    @IBOutlet weak var nameUserLabel: UILabel!
+    @IBOutlet weak var lastMessageLabel: UILabel!
+    @IBOutlet weak var hourLastMessageLabel: UILabel!
+    
+    private var friendMessage : MessagesUser?
+    static var NameCell = "MessageCell"
+    static var nibName = UINib(nibName: NameCell, bundle: nil)
+    
+    
+    func setFriendMessages(friend : MessagesUser){
+        friendMessage = friend
+        if let userFriend = friend.getInformationFriend(){
+            if let urlImage = URL(string: userFriend.imageURL){
```

Comment:
we should use a default image and also ... how is this image retrieved?? sync or async?